xapian-letor/api/featuremanager.cc

```diff
@@ -69,9 +69,9 @@ FeatureManager::getlabel(map<string, map<string, int> > qrel2, const Document &d
 }
 
 Xapian::RankList
-FeatureManager::create_rank_list(const Xapian::MSet & mset, std::string & qid)
+FeatureManager::create_rank_list(const Xapian::MSet & mset, std::string & qid, int TrainorTest)
```

Comment:
Formal parameters should be in lower-case, really (`trainor_test`); also, I'm not sure from the name what this does.